src/NetAnalyzers/Core/Microsoft.CodeQuality.Analyzers/Maintainability/AvoidDeadConditionalCode.cs

```diff
@@ -90,7 +91,9 @@ public override void Initialize(AnalysisContext context)
                             var cfg = operationBlockContext.GetControlFlowGraph(operationRoot);
                             var wellKnownTypeProvider = WellKnownTypeProvider.GetOrCreate(operationBlockContext.Compilation);
                             var valueContentAnalysisResult = ValueContentAnalysis.TryGetOrComputeResult(cfg, owningSymbol, wellKnownTypeProvider,
-                                    operationBlockContext.Options, AlwaysTrueFalseOrNullRule, operationBlockContext.CancellationToken,
+                                    operationBlockContext.Options, AlwaysTrueFalseOrNullRule,
+                                    PointsToAnalysisKind.Complete,
```

Comment:
This can potentially be switched to partial analysis in future, if required. Currently, this rule is disabled by default.